distribution/docker/src/docker/iron_bank/hardening_manifest.yaml

```diff
@@ -47,7 +47,7 @@ maintainers:
   - name: "Mark Vieira"
     email: "mark.vieira@elastic.co"
     username: "mark-vieira"
-  - name: "Rene Gröschke"
+  - name: "Rene Groschke"
```

Comment:
Can we make it `Rene Groeschke` then? :)

Reply:
Indeed, sorry 🤦🏻